artichoke-backend/src/extn/core/kernel/mruby.rs

```diff
@@ -17,6 +17,7 @@ pub fn init(interp: &mut Artichoke) -> InitializeResult<()> {
         .add_method("load", artichoke_kernel_load, sys::mrb_args_rest())?
         .add_method("print", artichoke_kernel_print, sys::mrb_args_rest())?
         .add_method("puts", artichoke_kernel_puts, sys::mrb_args_rest())?
+        .add_method("p", artichoke_kernel_p, sys::mrb_args_rest())?
```

Comment:
can this be alphabetized before `print` to match the Ruby docs:

https://ruby-doc.org/core-2.6.3/Kernel.html#method-i-p